web/src/OverviewActionBar.tsx

```diff
@@ -659,6 +660,88 @@ export function OverviewWidgets(props: { buttons?: UIButton[] }) {
     </WidgetRoot>
   )
 }
+const EnableDisableResourceRoot = styled(ButtonRoot)<{
+  confirmingDisable: boolean
+}>`
+  ${(props) =>
+    props.confirmingDisable &&
+    `
+    color: ${Color.white};
+    background-color: ${Color.red};
+
+    &:hover {
+      color: ${Color.white};
+      background-color: ${Color.red};
+    }
+  `};
+  margin-left: auto;
+  padding-left: ${SizeUnit(0.25)};
+`
+const EnableDisableResourceButtonContent = styled.div`
+  .MuiIcon-root {
+    font-size: ${FontSize.small};
+  }
+  display: flex;
+  align-items: center;
+`
```

Comment:
is there a reason why you need a `div` with these styles inside the `button`? (afaik, having a `div` inside a `button` isn't good practice, but i forget the exact html spec wording!) 

i think you could get away with styling the `Icon` directly with `font-size` and drop the `div` + display styles.

Reply:
ah yeah that's simpler, thanks!